pages/linux/snmpwalk.md

```diff
@@ -0,0 +1,24 @@
+# snmpwalk
+
+> SNMP query tool.
+> More information: <https://linux.die.net/man/1/snmpwalk>.
+
+- Query target system information using SNMPv1 and community string:
+
+`snmpwalk -v1 -c {{community}} {{ip}}`
+
+- Query specific target system information by OID using SNMPv2 on a specified port:
+
+`snmpwalk -v2c -c {{community}} {{ip}}:{{port}} {{oid}}`
+
+- Query specific target system information by OID using SNMPv3 and authentication without encryption:
```

Comment:
Perhaps this will read better?
```suggestion
- Query specific system information on a remote host by OID using SNMPv3 and authentication without encryption:
```